README.md

```diff
@@ -167,14 +167,6 @@ julia> ilr(c)
 
 and their inverses `alrinv`, `clrinv` and `ilrinv`.
 
-The package also defines transforms for tables following to the
-[TableTransforms.jl](https://github.com/JuliaML/TableTransforms.jl) interface, including `Closure`, `Remainder`, `ALR`, `CLR`, `ILR`.
-These transforms are functors that can be used as follows:
-
-```julia
-julia> table |> ILR()
-```
```

Comment:
Here we can point to the TableTransforms.jl package saying that these transforms are defined over there.